prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-presubmits.yaml

```diff
@@ -71,8 +71,7 @@ presubmits:
           readOnly: true
         resources:
           requests:
-            # When running at parallel=15, we use 20Gi. So request 80Gi to be safe.
-            memory: "80Gi"
+            memory: "108Gi"
```

Comment:
I think `108Gi` might not leave enough memory for other system pods to run on the node. Only `109.12Gi` is allocatable on the node, so this leaves just ~1 GB for other pods. Maybe we can try something around 100-105 GB?

Reply:
Sounds good. Updated the PR.